from src.calculadora_bugada import soma  # Este import falha

def test_soma_bugada():
    assert soma(2, 2) == 4